test/e2e_node/image_credential_provider.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2enode
+
+import (
+	"github.com/onsi/ginkgo"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/uuid"
+	"k8s.io/kubernetes/test/e2e/framework"
+	imageutils "k8s.io/kubernetes/test/utils/image"
+)
+
+var _ = SIGDescribe("ImageCredentialProvider [Feature: KubeletCredentialProviders]", func() {
```

Comment:
we do tags without spaces:

```suggestion
var _ = SIGDescribe("ImageCredentialProvider [Feature:KubeletCredentialProviders]", func() {
```

Once it's graduated to beta, since the feature is not optional and test brings its own creds provider, this tag can actually be changed to the NodeConformance. But this is for the next PR

Reply:
> Once it's graduated to beta, since the feature is not optional and test brings its own creds provider, this tag can actually be changed to the NodeConformance. But this is for the next PR

The feature gate is on by default, but the feature is still opt-in through kubelet flags/configuration.

Reply:
Also fixed the tag, thanks!